pom.xml

```diff
@@ -684,12 +684,11 @@
                             <includeModule>polly</includeModule>
                         </includeModules>
                         <excludes>
-                            <!-- TODO remove after release -->
-                            <exclude>software.amazon.awssdk.enhanced.dynamodb.extensions.annotations.DynamoDbVersionAttribute#incrementBy()</exclude>
-                            <exclude>software.amazon.awssdk.enhanced.dynamodb.extensions.annotations.DynamoDbVersionAttribute#startAt()</exclude>
                             <exclude>*.internal.*</exclude>
                             <exclude>software.amazon.awssdk.thirdparty.*</exclude>
                             <exclude>software.amazon.awssdk.regions.*</exclude>
+                            <!-- TODO remove after release -->
+                            <exclude>software.amazon.awssdk.awscore.interceptor.TraceIdExecutionInterceptor</exclude>
```

Comment:
japicmp is flagging this as a breaking change, however this is an SdkProtectedApi class, and the removal is of override methods whose implementation was not supported server side.